br.com.zoop.zooppcserver.metainfo.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>br.com.zoop.zooppcserver</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
```

Comment:
I don't see any indication that this is the license since there is not even a link to the source code.

Reply:
Is it necessary to be open source? I need to remove that license there.

Reply:
It must be accurate.